infra/modules/database/variables.tf

```
@@ -7,7 +7,12 @@ variable "name" {
   }
 }
 
-variable "access_policy_name" {
+variable "app_access_policy_name" {
+  description = "name of the IAM policy to create that will be provide the ability to connect to the database as a user that will have read/write access."
+  type        = string
+}
+
+variable "migrate_access_policy_name" {
```

nit: can we also rename this to migrator_access_policy_name
```suggestion
variable "migrator_access_policy_name" {
```

---

Replaced in separate commit including other instances and references